locales/th/json-inline.json

```diff
@@ -1,16 +1,16 @@
-    "The :attribute must be at least :length characters and contain at least one number.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัว และประกอบด้วยตัวเลขอย่างน้อยหนึ่งตัว",
-    "The :attribute must be at least :length characters and contain at least one special character and one number.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัว และประกอบด้วยอักขระพิเศษอย่างน้อยหนึ่งตัวและตัวเลขอย่างน้อยหนึ่งตัว",
-    "The :attribute must be at least :length characters and contain at least one special character.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัว และประกอบด้วยอักขระพิเศษอย่างน้อยหนึ่งตัว",
-    "The :attribute must be at least :length characters and contain at least one uppercase character and one number.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัว และประกอบด้วยตัวอักษรพิมพ์ใหญ่อย่างน่้อยหนึ่งตัว และตัวเลขอย่างน้อยหนึ่งตัว",
-    "The :attribute must be at least :length characters and contain at least one uppercase character and one special character.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัวอักษร และประกอบด้วยตัวอักษรพิมพ์ใหญ่อย่างน่้อยหนึ่งตัว และอักขระพิเศษอย่างน้อยหนึ่งตัว",
-    "The :attribute must be at least :length characters and contain at least one uppercase character, one number, and one special character.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัวอักษร และประกอบด้วยตัวอักษรพิมพ์ใหญ่อย่างน่้อยหนึ่งตัว, ตัวเลขหนึ่งตัว, และอักขระพิเศษหนึ่งตัวอักษร",
-    "The :attribute must be at least :length characters and contain at least one uppercase character.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัว และประกอบด้วยตัวอักษรพิมพ์ใหญ่อย่างน่้อยหนึ่งตัว",
-    "The :attribute must be at least :length characters.": "ช่องข้อมูลนี้ต้องมีอักขระอย่างน้อย :length ตัว",
-    "The :attribute must contain at least one letter.": "ช่องข้อมูลนี้ต้องประกอบด้วยตัวอักษรอย่างน้อยหนึ่งตัว",
-    "The :attribute must contain at least one number.": "ช่องข้อมูลนี้ต้องประกอบด้วยตัวเลขอย่างน้อยหนึ่งตัว",
-    "The :attribute must contain at least one symbol.": "ช่องข้อมูลนี้ต้องประกอบด้วยสัญลักษณ์อย่างน้อยหนึ่งตัว",
-    "The :attribute must contain at least one uppercase and one lowercase letter.": "ช่องข้อมูลนี้ต้องประกอบด้วยตัวอักษรพิมพ์ใหญ่อย่างน้อยหนึ่งตัว และตัวอักษรพิมพ์เล็กอย่างน้อยหนึ่งตัว",
-    "The given :attribute has appeared in a data leak. Please choose a different :attribute.": "ข้อมูลในช่องข้อมูลนี้ถูกพบในข้อมูลที่มีการรั่วไหล โปรดเลือกข้อมูลอื่น"
+    "The :attribute must be a valid role.": "ข้อมูลในฟิลด์นี้ต้องระบุบทบาทที่ถูกต้อง",
+    "The :attribute must be at least :length characters and contain at least one number.": "ข้อมูลในฟิลด์นี้ต้องมีความยาวอย่างน้อย :length ตัวอักษร และต้องมีตัวเลขอย่างน้อยหนึ่งตัว",
+    "The :attribute must be at least :length characters and contain at least one special character and one number.": "ข้อมูลในฟิลด์นี้ต้องมีความยาวอย่างน้อย :length ตัวอักษร และต้องประกอบด้วยอักขระพิเศษและตัวเลข อย่างละหนึ่งตัวเป็นอย่างน้อย",
```

Comment:
Hi, I can check on this one.
The sentence is correct because there is no sentence-end punctuation in Thai. That means a period (`.`) is not required.

In this case, `ง` is part of the word, for example, ต้อง means must.

Reply:
Ok, got it